Vision/Vision.bs

```diff
@@ -107,76 +99,75 @@ Boilerplate: omit conformance
 
 # Vision for W3C # {#vision-org}
 
-	The fundamental purpose of W3C is to provide an open forum
+	The World Wide Web Consortium (W3C) was founded as an organization 
+	to provide a consistent architecture 
+	across the rapid pace of progress in the Web, 
+	and to build a common community to support its development. 
+	It has become an association where diverse voices from around the world 
+	and from different industries and organizations work together to evolve the Web.
+
+	To build a better future, the W3C must rise even further 
+	to the challenge of improving the Web's fundamental integrity, 
+	while continuing to expand the Web’s scope and reach. 
+	The Operational Principles for the W3C should reflect the core values of the Web itself.
+
+	These core values will be clearly demonstrated by how W3C leads the Web forward: 
+	by being inclusive, principled, and continually striving to make the Web better 
+	through these principles and the Ethical Web Principles. 
+	As the Ethical Web Principles state, “The web should empower an equitable, informed and interconnected society.”
+
+# Operational Principles for W3C # {#op-principles}
+
+	The fundamental function of W3C is to provide an open forum
 	where diverse voices from around the world
 	and from different organizations and industries
 	work together to build consensus
 	on voluntary global standards for Web technologies.
+	In order to fulfill this function, we will follow these operational principles:
 
-	* We put the needs of users first:
+	* W3C intentionally involves stakeholders from end to end 
+		in building the Web: developers, content creators, and end users. 
+		Our work will not be dominated by any person, company, or interest group, 
+		but uses consensus from the community to 
+		put the needs of users first: 
 		above authors, publishers, implementers, paying W3C Members, or theoretical purity.
-	* This organization believes in diversity
+	* We believe in diversity
 		and inclusion of participants from different
 		geographical locations,
 		cultures,
 		languages,
 		disabilities,
 		gender identities,
+		industries,
+		organizational sizes,
 		and more.
-	* We believe in principled, community-wide consensus-building
-		as the basis for building standards.
```

Comment:
Not sure this needed to be here, as I think it's covered in the first point.

Reply:
I think it's better as its own point here. It gets lost up there, as if it's merely a tool for putting users first -- but it's more than just a tool for that purpose, it's a core value on its own.

Putting users first isn't the reason we operate by consensus. We do it because we don't want the Web to be dominated by any one party, and because the process of getting to consensus brings out diverse perspectives and intense reviews that lead to a higher quality of technology (more thorough, more thoughtful, more cross-platform, more long-lived, etc.).

Reply:
Re-added.